src/components/Grid/NoRowsOverlay.tsx

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
This is not necessary, display "No data" in text and remove this file

Reply:
Yes, to remove